ldes-server-port-admin-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminShapeRestControllerTest.java

```diff
@@ -44,13 +49,17 @@
 @ActiveProfiles({ "test", "rest" })
 @ContextConfiguration(classes = { AdminShapeRestController.class,
 		AdminWebConfig.class, AdminRestResponseEntityExceptionHandler.class })
+@RecordApplicationEvents
 class AdminShapeRestControllerTest {
 	@MockBean
 	private LdesConfigModelService ldesConfigModelService;
 	@MockBean
 	@Qualifier("shapeShaclValidator")
 	private LdesConfigShaclValidator ldesConfigShaclValidator;
 
+	@Autowired
+	private ApplicationEvents applicationEvents;
```

Comment:
Ha, that's a secret trick. I made a suggestion using a captor: https://www.baeldung.com/mockito-argumentcaptor